packages/components/package.json

```diff
@@ -28,7 +28,7 @@
     "start": "stencil build --watch --docs-readme",
     "build": "stencil build --docs-readme",
     "clean": "rimraf www dist loader",
-    "test": "npm run unit",
+    "test": "pnpm run unit",
```

Comment:
This does not necessarily need to be a part of this PR, I just encountered an error with the pnpm bootstrap, and it could be fixed with using pnpm instead of npm.